spec/ddtrace/contrib/sinatra/activerecord_spec.rb

```diff
@@ -100,6 +100,17 @@ def migrate_db
       expect(sqlite_span.span_type).to eq(Datadog::Ext::SQL::TYPE)
       expect(sqlite_span).to_not have_error
       expect(sqlite_span.parent_id).to eq(route_span.span_id)
+      expect(sqlite_span.get_tag(Datadog::Ext::Metadata::TAG_COMPONENT))
+        .to eq(Datadog::Contrib::ActiveRecord::Ext::TAG_COMPONENT)
+      expect(sqlite_span.get_tag(Datadog::Ext::Metadata::TAG_OPERATION))
+        .to eq(Datadog::Contrib::ActiveRecord::Ext::TAG_OPERATION_SQL)
+      expect(sqlite_span.get_tag(Datadog::Ext::Metadata::TAG_PEER_SERVICE))
+        .to eq('sqlite')
+
+      unless adapter_host.nil?
+        expect(sqlite_span.get_tag(Datadog::Ext::Metadata::TAG_PEER_HOSTNAME))
+          .to eq(adapter_host.to_s)
+      end
```

Comment:
Should we also test that no tag is set when `adapter_host` is `nil`? Also, since we control the tests, shouldn't the `adapter_host` be deterministic? This test makes it sound like it can sometimes be `nil`, and others not?